fixtures/generic/rgbw-bar-8.json

```
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "RGBW Bar 8",
+  "categories": ["Pixel Bar"],
```

```suggestion
  "categories": ["Color Changer"],
```
It's a row of lights but there's no individual pixel control.